src/AccessibilityInsights.SharedUx/Utilities/ExtensionMethods.cs

```diff
@@ -289,5 +289,19 @@ internal static BitmapSource ConvertToSource(this Bitmap bitmap)
             NativeMethods.DeleteObject(hbmp);
             return result;
         }
+
+        /// <summary>
+        /// Check whether p is fully visible based on current screen size
+        /// </summary>
+        /// <param name="p"></param>
+        /// <param name="c"></param>
+        /// <param name="margin"></param>
+        /// <returns></returns>
+        internal static bool IsVisibleLocation(this Rectangle p)
```

Comment:
This method was copied from lines 110-122 of https://github.com/microsoft/axe-windows/blob/master/src/Desktop/Utility/ExtensionMethods.cs. I intentionally didn't add a "using System.Windows.Forms" to the file, since it introduces class ambiguity that would then need to be addressed.